src/main/java/com/epam/ta/reportportal/commons/querygen/QueryBuilder.java

```diff
@@ -152,11 +155,16 @@ public QueryBuilder with(int limit) {
 	 * @return QueryBuilder
 	 */
 	public QueryBuilder with(Sort sort) {
+
+		Set<Pair<String, Sort.Direction>> sortingSelect = Sets.newHashSet();
```

Comment:
Can you explain why do wee need it?

Reply:
We select fields from 'order by' statement of the sub-query, to use ordering in the outer query, and API allows us to create a filter with orderings for the same field (ASC + DESC), so when we select the same field twice, ambiguous ordering column error occurres

Reply:
How was it resolved in the 4'th version? Maybe it can be checked somewhere upper ?

Reply:
It can be handled only on this level, because it's the only place from which 'query' is available